### DISCUSSION TITLE
Filtering posts without alt text

### DISCUSSION THREAD

**Discussion by markwyner** [mastodon/mastodon#27487] [General]
Hey, all.

Alt text is super important to the Mastodon community. Someone on my timeline was wondering if they could filter their feed so images with no alt text would be omitted/hidden. That’s a really interesting idea that would, in theory, help promote usage of alt text.

I believe it should be worth considering as a setting. How do folks feel about that?

**Comment by Monikae** [comment]
Two of my blind mutuals have asked for this feature within the last 5 days.

**Comment by roycewilliams** [comment]
Beyond accessibility (an excellent goal), the other reason this feature is needed is that Mastodon's filters also operate on alt text - so images *without* alt text *bypass filters*. So all of the important reasons that people use filters are automatically brought into the scope of impact of the lack of this feature.

Once this feature is added, an important loophole will be closed in Mastodon's otherwise superior filtering capabilities.

See also related issues: 
* #26671
* #18955
